vendor/modules.txt

```diff
@@ -51,6 +51,41 @@ github.com/aokoli/goutils
 github.com/armon/go-metrics
 # github.com/avast/retry-go v2.2.0+incompatible
 github.com/avast/retry-go
+# github.com/aws/aws-sdk-go v1.24.1
+github.com/aws/aws-sdk-go/aws
```

Comment:
Hmm?

Reply:
go module clean up. maybe QA added in last comment but forgot to run go module